src/formatter.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+// This file is copied from eslint-formatter-pretty except that it also outputs
+// the source line that failed the rules because we don't keep track of line/col numbers
+
+const path = require('path');
+const chalk = require('chalk');
```

Comment:
Did you consider [`ansi-colors`](https://www.npmjs.com/package/ansi-colors) instead? I reached for `chalk` in another experiment and saw that `ansi-colors` had no dependencies.

Reply:
Thanks for reminding me 👍